requirements/required.txt

```diff
@@ -9,7 +9,7 @@ lightly==1.5.12
 lightning[pytorch-extra]==2.4.0
 matplotlib==3.9.2
 numpy==2.1.1
-pandas==2.2.2
+pandas[parquet]==2.2.2
```

Comment:
Let's undo this change so we can test that the tests also pass even if optional dependencies like pyarrow aren't installed.

Reply:
I'm not understanding what exactly you are looking for here.
- We need pandas[parquet] for tests for the fields of the world dataset to pass
- If we have pandas<2 for min-reqs then we need pyarrow for the min tests to pass

Reply:
Correct. But TorchGeo has required dependencies and optional dependencies. Although pandas is required, `pandas[parquet]` is not. The tests should all pass regardless of whether or not optional dependencies are installed. For example, we now have 8 datasets which require h5py. But those tests don't fail if h5py isn't installed, because we use:
```python
pytest.importorskip('h5py', minversion='3.6')
```
in all of those tests. We can do the same thing for pyarrow since it's only required by one dataset.

P.S. Didn't realize this number was now up to 8, maybe h5py should be required...